consumatio/external/tmdb.py

```diff
@@ -29,6 +32,7 @@ def get_movie_details(self: object, movie_id: int) -> dict:
         :param movie_id: <int> Id of the movie to fetch details for
         :return: <dict> Movie details
         """
+        logger.info("Request:'movie_details' transmitted to database")
```

Comment:
Not in all cases something is transmitted to the database. In general, API requests are being made here. You should rather log database-related things in the corresponding get_data() method.